tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@oclif/tslint"
+  "extends": "@oclif/tslint",
+  "rules": {
+    "no-string-literal": false
```

Comment:
? afaict this is for https://github.com/heroku/heroku-pg-privatelink-cli/pull/12/files#diff-a3e0ec75733d1171c25f79b1b85e0780R14 but why not just `res[0].addon.name`?